alma/lib/Helpers/ClientHelper.php

```diff
@@ -147,18 +149,21 @@ public function getPaymentByTransactionId($transactionId)
     }
 
     /**
-     * @param string $orderExternalId
-     * @param array $data
+     * @param string $paymentId
+     * @param string $merchantOrderReference
+     * @param string $status
+     * @param bool|null $isShipped
      *
-     * @return null
+     * @return void
      *
-     * @throws Alma\API\Exceptions\ParametersException
-     * @throws Alma\API\Exceptions\RequestException
      * @throws ClientException
+     * @throws ParametersException
+     * @throws RequestError
+     * @throws RequestException
      */
-    public function sendOrderStatus($orderExternalId, $data)
+    public function sendOrderStatus($paymentId, $merchantOrderReference, $status, $isShipped = null)
     {
-        $this->getClientOrdersEndpoint()->sendStatus($orderExternalId, $data);
+        $this->getClientPaymentsEndpoint()->addOrderStatusByMerchantOrderReference($paymentId, $merchantOrderReference, $status, $isShipped);
```

Comment:
It will be good to use the new ClientModel instead current ClientHelper, but this function it was already in the clientHelper. Maybe we can add @deprecated on each getClient... function in clientHelper

Reply:
Need a deep refactoring for php-client